packages/web/src/routes.ts

```diff
@@ -28,6 +28,10 @@ const commanderByID = (race = ":race", commanderID = ":commanderID") => {
   return `${commanderList(race)}/${commanderID}`;
 };
 
+const desktopAppBase = () => {
+  return "/desktopApp";
```

Comment:
I think it really doesn't really matter how the url looks like - all the different things are used. But I would prefer for it to be `desktop-app`